sonar-plugin-api/src/main/java/org/sonar/api/measures/RuleMeasure.java

```diff
@@ -31,7 +31,9 @@
 
 /**
  * @since 1.10
+ * @deprecated since 5.2. Ignored by design because of Compute Engine.
  */
+@Deprecated
 public class RuleMeasure extends Measure {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)